packages/strapi-helper-plugin/lib/src/components/PageFooter/index.js

```diff
@@ -54,6 +55,7 @@ function PageFooter(props) {
 }
 
 PageFooter.defaultProps = {
+  context: {},
```

Comment:
You don't need to pass the context in props.
You should use the following instead

```
PageFooter.contextTypes = {
 emitEvent: PropTypes.func,
};
```

Reply:
Nop, because in this component the context is available through the props (pure compo) and it was able to get it using your code.